### PR TITLE
add docker image and k8s manifest for loadgen

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Instead of having the user to execute the loadgenerator locally, the user can deploy the docker image to run constant load on the carts service in all three namespaces.

```
kubectl apply -f cartsloadgen.yaml 
```

### CODE REVIEW

**Comment by jetzlstorfer** [comment]
We can still keep the go binaries to support older use cases described in the docs section but going forward I would suggest removing the go-files and do the loadgeneration from within the cluster to have a constant load on the services.

**Comment by jetzlstorfer** [comment]
I've now added some ENV variables to be able to adjust the ITEM_ID, SLEEP_TIME, and STAGES that are load tested.

File: load-generation/cartsloadgen/deploy/cartsloadgen-base.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cartsloadgen
+    spec:
+      containers:
+      - name: cartsloadgen
+        image: docker.io/jetzlstorfer/cartsloadgenerator:0.1
```
Comment (christian-kreuzberger-dtx) [inline load-generation/cartsloadgen/deploy/cartsloadgen-base.yaml:21]: Please use `docker.io/keptnexamples...`

File: load-generation/cartsloadgen/generate_traffic.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+echo "Press [CTRL+C] to stop.."
+
+echo "Using this domain: $KEPTN_DOMAIN"
+stages=( $CARTS_STAGES )
+echo $stages
+
+# Declare variables
+contentType="Content-Type: application/json"
+dtHeaderGet="x-dynatrace-test: LSN=generate_traffic.sh;LTN=Traffic generation;TSN=Get Items in cart;"
+dtHeaderAdd="x-dynatrace-test: LSN=generate_traffic.sh;LTN=Traffic generation;TSN=Get Items in cart;"
```
Comment (christian-kreuzberger-dtx) [inline load-generation/cartsloadgen/generate_traffic.sh:12]: Please change `TSN` to `Add item...`

Comment (jetzlstorfer) [reply load-generation/cartsloadgen/generate_traffic.sh]: changed

File: load-generation/cartsloadgen/deploy/cartsloadgen-base.yaml (hunk 1 above)
Comment (jetzlstorfer) [reply load-generation/cartsloadgen/deploy/cartsloadgen-base.yaml]: updated

**Pull request merged**